app/src/main/java/ar/com/wolox/android/training/ui/home/HomeFragment.java

```diff
@@ -0,0 +1,27 @@
+package ar.com.wolox.android.training.ui.home;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+
+/** My <b>HomeFragment</b>. */
+public class HomeFragment extends WolmoFragment<HomePresenter> implements IHomeView {
+
+    private static HomeFragment instance;
+
+    public static HomeFragment newInstance() {
```

Comment:
I will leave you here 2 links about fragments and activities instantiation. they are not a MUST but they are some good practices. 
https://medium.com/@azjkjensen/using-the-newinstance-pattern-in-kotlin-e40c1b4ba1ef
https://hackernoon.com/object-oriented-tricks-4-starter-pattern-android-edition-1844e1a8522d

We can go further on this later by call but as for now let's try to avoid the use of singletons

Reply:
Thanks for the info and the help.
I think I implement them both well.
Please check it out and tell me 😁